etcdserver/etcdhttp/httptypes/errors.go

```diff
@@ -43,7 +43,10 @@ func (e HTTPError) WriteTo(w http.ResponseWriter) {
 	if err != nil {
 		plog.Panicf("marshal HTTPError should never fail (%v)", err)
 	}
-	w.Write(b)
+	if _, err := w.Write(b); err != nil {
+		plog.Errorf("http write errors with HTTPError (%v)", err)
```

Comment:
Also we probably should make this a debug level. A client side error can cause this too. This is not a server side error in many cases.